### ISSUE TITLE
Does anyone have experience using this with Paypal's Bootstrap Accessibility (missing ARIA roles)

### ISSUE THREAD

**Issue by CoralSilver** [silviomoreto/bootstrap-select#1222]
1) Has anyone successfully integrated this with [Paypal's Bootstrap Accessibility plugin] (https://github.com/paypal/bootstrap-accessibility-plugin)? That plugin makes dropdowns more accessible, but I am finding that dropdowns with bootstrap select do not display correctly once an option has been selected [EXAMPLE] (http://plnkr.co/edit/XwCVxyq20A71z1G8oakZ?p=preview).

2) Bootstrap-select is missing required ARIA roles.  It adds role=menu on the` <ul>` but it should also be adding `role="menuitem"` to the `<a>` and `role="presentation"` to the `<li>`.  Bootstrap accessibility adds this normally to dropdowns, but the DOM structure for bootstrap-select is different.

**Comment by caseyjhol** [comment]
Related to #541.
